hedera-node/hedera-config/src/main/java/com/hedera/node/config/sources/PropertyConfigSource.java

```diff
@@ -88,6 +89,17 @@ public String getValue(@NonNull String key) throws NoSuchElementException {
         return properties.getProperty(key);
     }
 
+    @Override
+    public boolean isListProperty(@NonNull final String propertyName) throws NoSuchElementException {
+        return false;
+    }
+
+    @NonNull
+    @Override
+    public List<String> getListValue(@NonNull final String propertyName) throws NoSuchElementException {
+        return List.of();
```

Comment:
Same here

Reply:
Currently, we're focusing on the YAML source, but we've created issues to add support for the new list properties in other sources as well. For now, the other sources continue to work in the original way.